examples/demo/src/visitors/VisitorEdit.js

```diff
@@ -12,7 +12,7 @@ import {
     ReferenceManyField,
     TabbedForm,
     TextField,
-    TextInput,
```

Comment:
I see only coding style changes in this file, you didn't add a way to update the password?